.circleci/config.yml

```
@@ -99,20 +101,22 @@ jobs:
           args: jacocoIntegrationTestReport
       - codecov/upload:
           flags: integration
-  publish:
+  merge-publish:
     executor: gradle_docker
     steps:
       - setup_build_environment
-      - setup_remote_docker
+      - setup_remote_docker: *latest_remote_docker
       - docker_login
-      - gradle:
-          args: :tag -Prelease
-      - add_ssh_keys:
-          fingerprints:
-            - '21:f0:c0:3c:a3:c1:4f:9b:69:d8:1a:53:aa:f1:b1:57'
-      - run: git push origin $(./gradlew -q :printVersion)
       - gradle:
           args: dockerPushImages
+  release-publish:
+    executor: gradle_docker
+    steps:
+      - setup_build_environment
+      - setup_remote_docker: *latest_remote_docker
+      - docker_login
+      - gradle:
+          args: publish dockerPushImages
```

No publish task previously
```suggestion
          args: dockerPushImages
```